### ISSUE TITLE
Docs index shows as empty in local builddocs

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#2835]

**Comment by andy-maier** [comment]
The reason is related to the difference in the build directory tree:

- RTD: Files are processed without "docs"
- make builddoc: Files are processed with "docs"

Not clear where that difference comes from, though.

**Comment by andy-maier** [comment]
From the file names reported by Sphinx on RTD it becomes clear that Sphinx runs in the "docs" directory on RTD.

However, there are some strange things around this:
* The debug print of the current directory in conf.py shows "pywbem/docs" in both RTS and local builds. I would have expected that local builds show only "pywbem".
* The log files on RTD do not show anything about switching directories. That must happen between "cat docs/conf.py" and the first sphinx build command, but does not show up in the log file, not even in its raw version.

Also, RTD seems to append some code to the conf.py file before showing it in the log file. The RTD log file does not reveal where that happens. I looked at the additional code, but could not find anything that would explain the directory switch or otherwise switching to docs.

I tried building locally from within the docs directory, and that successfully generated the index file, plus it allowed to get rid of the distinction for RTD vs local when setting the "master"doc" variable in conf.py. That solution is in PR #2837 .
